images/bootstrap/gcs_restapi.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+# Copyright 2021 The KubeVirt Authors.
```

Comment:
Was this copied from a kubernetes repo? or is this copyright just copied from another script? 

```suggestion
# Copyright 2024 The KubeVirt Authors.
```

Reply:
Yeah I think we can update this as its a new file.

Reply:
Thanks, Updated.